PlaySpotApi/Routes/LocationRoutes.cs

```diff
@@ -55,6 +59,74 @@ public static RouteGroupBuilder MapLocationRoutes(this RouteGroupBuilder group)
             .Produces<List<Location>>(StatusCodes.Status200OK)
             .Produces(StatusCodes.Status404NotFound);
 
+
+            //this get request returns all the locations for a given sport with a specified radius.
+            group.MapGet("/locations-by-sport-", async (
```

Comment:
We need to decide on a pattern to use for all endpoints. 

I think this one should look like `/locations/by-sport?sportName=&coordinates=&radius=` 
or
It could look like `/locations/by-sport/sportName={sportName}&coordinates={coordinates}&radius={radius}

either way, make sure it isn't `/locations/locations-...` since everything in this class is at `/locations`